specification/deviceregistry/DeviceRegistry.Management/properties/ns_assetProperties.tsp

```diff
@@ -181,13 +188,43 @@ model NamespaceDatasetDataPoint is NamespaceDataPoint {
 @added(Versions.v2025_07_01_preview)
 model NamespaceEventDataPoint is NamespaceDataPoint;
 
+@doc("Defines the event group properties.")
+@added(Versions.v2025_10_01)
+model NamespaceEventGroup {
+  @doc("The name of the event group.")
+  name: string;
+
+  @doc("The address of the notifier of the event group in the asset (e.g. URL) so that a client can access the event group on the asset.")
```

Comment:
"Reference to a typed or untyped data source of events in the event group." 

The event notifier is still at event level, unless we want to also have it at group level to define one that applies to all events (similar to all other cascading definitions), then we would want to keep the name though ("EventNotifier").  See next comment